docs/otter_assign/v1/notebook_format.rst

```diff
@@ -98,25 +98,27 @@ their values:
         show_hidden: true
 
 You can also set the autograder up to automatically upload PDFs to student submissions to another 
-Gradescope assignment by setting the necessary keys in the ``pdfs`` subkey of ``generate``:
+Gradescope assignment by setting the necessary keys under ``generate``:
 
 .. code-block:: yaml
 
     generate:
-        token: ''
+        token: Glsdxkjf0900Qiz # optional
```

Comment:
I assume this is a fake token? Maybe use something like `YOUR_TOKEN_HERE` instead?

Reply:
Updated! I used the real-looking fake token to be in line with the course_id and assignment_id just below, but I changed it to just a place holder text now